cubeviz/layout.py

```diff
@@ -120,6 +126,54 @@ def __init__(self, session=None, parent=None):
         self._single_image = False
         self.ui.button_toggle_image_mode.setText('Single Image Viewer')
 
+    def _init_option_buttons(self):
+        self._option_buttons = [
+            self.ui.view_option_button,
+            self.ui.cube_option_button
+        ]
+
+        view_menu = self._dict_to_menu(OrderedDict([
+            ('Something', lambda: None),
+            ('Anything', lambda: None),
+            ('Testing', lambda: None)
```

Comment:
Do we want to merge this with these placeholders in? (fine by me as long as we fix it later!)

Reply:
I am going to merge with them in there.  I will be doing a quick turn around on another set of tweaks (smaller) now that this one is ready. During that one I can clean this up a bit.  ( @drdavella )

Reply:
But thank you for noting...